firebase/functions/index.js

```diff
@@ -0,0 +1,50 @@
+// @license
+// Copyright (c) 2017 Google Inc. All rights reserved.
+// This code may only be used under the BSD style license found at
+// http://polymer.github.io/LICENSE.txt
+// Code distributed by Google as part of this project is also
+// subject to an additional IP rights grant found at
+// http://polymer.github.io/PATENTS.txt
+
+const functions = require('firebase-functions');
+const request = require('request-promise-native');
+require('babel-polyfill');
```

Comment:
What requires this? Just curious

Reply:
Object.entries() IIRC. Firebase is on a pretty old node (6.14).